OrbitCore/Capture.h

```diff
@@ -63,13 +63,14 @@ class Capture {
   static std::shared_ptr<SamplingProfiler> GSamplingProfiler;
   static std::shared_ptr<Process> GTargetProcess;
   static std::shared_ptr<Preset> GSessionPresets;
-  static std::shared_ptr<CallStack> GSelectedCallstack;
+  static std::shared_ptr<HashedCallstack> GSelectedCallstack;
```

Comment:
I think this global is only used in CaptureWindow and there just in two lines right next to each other. I am in favor of removing it.

Reply:
Yes, and the same for Capture::GCallstacks; probably, I should remove both.